libslub/slub/obj.py

```diff
@@ -80,6 +80,9 @@ def to_string(self, name="", verbose=0, use_cache=False, indent=0, colorize_func
         else:
             t = "F"
         address = "{:#x}".format(self.address)
+        if self.address < 0xffff800000000000:
```

Comment:
I _think_ we don't currently restrict use of this library to x64 anywhere, so I would say this should be abstracted and per-architecture, otherwise this would break it on anything but x64.